### DISCUSSION TITLE
Corkey single use not perm?

### DISCUSSION THREAD

**Discussion by v4pl** [3kh0/ext-remover#951] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
So I use corkey but after i restart my chroembook it doesn't work anymore so i'm just wondering if it's perm.

**Comment by v4pl** [comment]
@AshtonDavies

**Reply by AshtonDavies** [reply, marked as answer]
It's not permanent. Any time you restart Chrome or Chrome OS, it will likely try to fix any corrupted extensions that were interrupted upon installation.

You just have to perform Corkey over again if your Chromebook restarts, and I know it probably sucks.

**Reply by v4pl** [reply]
Any perm exploit like Skiovox you know of for 121?

**Reply by AshtonDavies** [reply]
> Any perm exploit like Skiovox you know of for 121?

Even SKIOVOX has to be done every time you sign out or restart though.

**Reply by v4pl** [reply]
You do know that Skiovox prevent you from sign out after closing the lid

**Reply by AshtonDavies** [reply]
> You do know that Skiovox prevent you from sign out after closing the lid

Yes, because you're not sign in under your student policy. Not everyone has that enforced. I think some admins do that if you have to keep school devices at school.

**Reply by v4pl** [reply]
I'm just saying Skiovox was the best exploit so I'm now so frustrated that my caub didn't work.

**Reply by AshtonDavies** [reply]
> I'm just saying Skiovox was the best exploit so I'm now so frustrated that my caub didn't work.

I just hope someone finds a new unenrollment method for you all, such as Sh1mmer, or something similar to SKIOVOX, but I think both are unlikely.

**Reply by v4pl** [reply]
Is defog still usable? And I don't really want to unenroll cause of tests and everything.

**Reply by AshtonDavies** [reply]
> Is defog still usable? And I don't really want to unenroll cause of tests and everything.

It's been quite a while. I don't think it works due to the downgrade patch for the recovery repair screen, but I have not defogged anyone's Chromebooks since below that version.

**Comment by KandyKaneMarvel** [comment]
Well if it goes away when you restart then obviously not.

**Reply by v4pl** [reply]
I don't know if i'm doing it correctly and astrodavid said that you have to do something and it will stay
